assets/templates/search.tmpl

```diff
@@ -1,24 +1,34 @@
 {{ $lang := .Language }}
 <div class="page-content border">
-    <div class="wrapper">
-        <section class="col">
-            {{ template "partials/summary" . }}
-        </section>
-        <section>
-            <div class="col col--md-12 col--lg-18 padding-top-md--2 padding-right-md--1 padding-bottom-md--4 padding-left-sm--1 padding-left-md--1 flush js-mobile-filters__title">
-                <h3 class="inline-block flush font-size--18">{{ localise "FilterResults" $lang 4 }}</h3>
-            </div>
-            <div class="col-wrap col--md-34 col--lg-40">
-                {{ template "partials/sort" . }}
-            </div>
-        </section>
-        <section class="col col--md-12 col--lg-18 margin-bottom">
-            {{ template "partials/filter" . }}
-        </section>
-        <section class="col col--md-34 col--lg-40 margin-left-md--1" aria-label="Search results">
-            {{ template "partials/department" . }}
-            {{ template "partials/results" . }}
-            {{ template "partials/pagination" . }}
-        </section>
-    </div>
+  <div class="search__container wrapper">
+    <section class="col search__summary">
+      {{ template "partials/summary" . }}
+    </section>
+    <section class="col col--md-12 col--lg-18 margin-bottom search__filter">
+      <div
+        class="col col--md-12 col--lg-18 padding-top-md--2 padding-right-md--1 padding-bottom-md--4 padding-left-sm--1 padding-left-md--1 js-mobile-filters__title"
+      >
+        <h3 class="font-size--18">
+          {{ localise "FilterResults" $lang 4 }}
+        </h3>
+      </div>
+      <div class="search__filter__content col--border-top">
+        {{ template "partials/filter" . }}
+      </div>
+    </section>
+
+    <section
+      class="col col--md-34 col--lg-40 margin-left-md--1"
+      aria-label="Search results"
+    >
```

Comment:
Could you move the chevrons up to the end of line? Both here and line 10.

Reply:
fixed